vnext/Microsoft.ReactNative/Views/ControlViewManager.cpp

```diff
@@ -124,7 +128,7 @@ void ControlViewManager::OnPropertiesUpdated(ShadowNodeBase *node) {
 
   // If developer specifies either the accessible and focusable prop to be false
   // remove accessibility and keyboard focus for component.
-  const auto isTabStop = (node->IsAccessible() && node->IsFocusable());
+  const auto isTabStop = node->IsDisable() ? false : node->IsAccessible() && node->IsFocusable();
```

Comment:
nit: could use bang operator here
```suggestion
  const auto isTabStop = !node->IsDisable() : node->IsAccessible() && node->IsFocusable();
```